ipvs/ipvs.go

```diff
@@ -62,6 +62,7 @@ type Destination struct {
 	LowerThreshold      uint32
 	ActiveConnections   int
 	InactiveConnections int
+	Stats               SvcStats
```

Comment:
this could be `DstStats` (instead of `SvcStats`).

Maybe like so:
```
type DstStats SvcStats
```

Although it seems redundant, since the kernel seems to report them using the same datastructure, so no point creating 2 go datastructures to represent the same kernel datastructure. (Other than to give them prettier names).